README.md

```diff
@@ -1,5 +1,13 @@
 # vinum
 
+<p align="center">
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="./resources/logo/logo-dark-256.png">
+    <source media="(prefers-color-scheme: light)" srcset="./resources/logo/logo-light-256.png">
+    <img alt="Default vinum logo" src="./resources/logo/logo-dark-256.png">
+  </picture>
+</p>
```

Comment:
I think I prefer the `logo-light` even for dark themes. I'm not a fan of white brackets XD. I will a make a poll on telegram for deciding that
![image](https://github.com/user-attachments/assets/56a85f21-063a-480d-bb98-8a2245a8c6a0)

Reply:
Apparently, I was wrong XD